content/applications/finance/accounting/fiscal_localizations/localizations/united_kingdom.rst

```diff
@@ -0,0 +1,84 @@
+==============
+United Kingdom
+==============
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the following modules to get all the features of the UK
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`UK - Accounting`
+     - `l10n_uk`
+     - Installing this module grants you access to:
+        - CT600-ready chart of accounts
+        - VAT100-ready tax structure
+        - InfoLogic UK counties listing
+        - a few other adaptations
+   * - :guilabel:`UK - Accounting Reports`
+     - `l10n_uk_reports`
+     - Installing this module grants you access to:
+        - Accounting reports for UK
+        - Allows sending the tax report via the MTD-VAT API to HMRC.
+
+.. tip::
+   Installing the module :guilabel:`UK - Accounting Reports` installs all two modules at
+   once.
+
+.. note::
+   Your company should be located in the UK to be able to submit HMRC reports.
+
+.. seealso::
+   - `HM Revenue & Customs <https://www.gov.uk/government/organisations/hm-revenue-customs/>`_
+   - `Overview of Making Tax Digital <https://www.gov.uk/government/publications/making-tax-digital/overview-of-making-tax-digital/>`_
+
```

Comment:
```suggestion

```
Might be worth adding a section here on the UK - Accounting module, and more specifically *what* it includes. Since in the task people seem to ask for screenshots, you could maybe add a chart of accounts of a UK local, as well as the tax structure (and where to find them)
![uk-accounts](https://user-images.githubusercontent.com/104567387/209557765-c7c10a9e-e344-456c-9a64-95e6208a4d71.png)

Reply:
Updated. New sections added.